app/routes/forum/components/ForumEditor.tsx

```diff
@@ -84,7 +84,7 @@ const ForumEditor = () => {
         {({ handleSubmit }) => (
           <Form onSubmit={handleSubmit}>
             <Field
-              placeholder="Title"
+              placeholder="Tips til utveksling, hvordan velge veileder eller er indøk virkelig homo?"
```

Comment:
🤔

Reply:
Yeah, maybe rephrase this?

Reply:
oh yea that wasn't _really_ supposed to get added 😆